pkgs/misc/drivers/logiops/0_3_3.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  glib,
+  udev,
+  libevdev,
+  libconfig,
+}:
+
+stdenv.mkDerivation (
+  oldAttrs: {
+    pname = "logiops";
+    version = "0.3.3";
+
+    src = fetchFromGitHub {
+      owner = "PixlOne";
+      repo = "logiops";
+      rev = "v${oldAttrs.version}";
+      sha256 = "sha256-9nFTud5szQN8jpG0e/Bkp+I9ELldfo66SdfVCUTuekg=";
+      # In v0.3.0, the `ipcgull` submodule was added as a dependency
+      # https://github.com/PixlOne/logiops/releases/tag/v0.3.0
+      fetchSubmodules = true;
+    };
+
+    nativeBuildInputs = [
+      cmake
+      pkg-config
+    ];
+    buildInputs = [
+      udev
+      libevdev
+      libconfig
+      glib
+    ];
+
+    cmakeFlags = [
+      "-DLOGIOPS_VERSION=${oldAttrs.version}"
+      "-DDBUS_SYSTEM_POLICY_INSTALL_DIR=${placeholder "out"}/share/dbus-1/system.d"
+      "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
Can this have a comment? Nix should be setting this on its own, corresponding to whether there's an `enableDebugging` wrapper used or not.

Reply:
I'm not actually 100% sure if this *needs* to be there; I simply added it because it was added in `logiops`' README from v0.2.3 &rarr; v0.3.3.

Reply:
[It is not needed as Nix already sets this](https://github.com/ckiee/nixpkgs/blob/ac47cd87a9fb6cf4bfeabbc9043f2df0b8e50ed2/pkgs/by-name/cm/cmake/setup-hook.sh#L110-L111)